skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillDRO.java

```diff
@@ -0,0 +1,29 @@
+package com.RDS.skilltree.Skill;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SkillDRO {
+    private String name;
+    private SkillType type;
+    private UUID createdBy;
+
+
+    static SkillModel toModel(SkillDRO skillDRO) {
```

Comment:
Is this method missing an access modifier? Or is it by design?

Reply:
It got missed, let me add it